### PR TITLE
Fixed Invite New User endpoints for organizations and projects

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Description

Bug: The Request Body Parameters `ID`, `OrgID`, `GroupID` must be blank when using `POST`, `PATCH` endpoints

Link to any related issue(s): 

## Type of change:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: mongodbatlas/organization_invitations.go [hunk 1]
```diff
@@ -102,12 +102,12 @@ func (s *OrganizationsServiceOp) Invitation(ctx context.Context, orgID, invitati
 // InviteUser invites one user to the Atlas organization that you specify.
 //
 // See more: https://docs.atlas.mongodb.com/reference/api/organization-create-one-invitation/
-func (s *OrganizationsServiceOp) InviteUser(ctx context.Context, invitation *Invitation) (*Invitation, *Response, error) {
-	if invitation.OrgID == "" {
```
Comment (gssbzn) [inline mongodbatlas/organization_invitations.go:106]: guessing this must be blank when POSTing, right? if so can you please update the description of the PR to include the why of the change?

**Review by gssbzn — APPROVED** [review]
LGTM

File: mongodbatlas/organization_invitations.go (hunk 1 above)
Comment (andreaangiolillo) [reply mongodbatlas/organization_invitations.go:106]: got the same issue with the update endpoints

Comment (andreaangiolillo) [reply mongodbatlas/organization_invitations.go:106]: updated

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**